extension/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I think that you need to depict  why this commit is to be merged by appending commit description enough.